knative/deployer_test.go

```diff
@@ -3,8 +3,31 @@ package knative
 import (
 	"os"
 	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	fn "knative.dev/kn-plugin-func"
 )
 
+func Test_setHealthEndpoints(t *testing.T) {
```

Comment:
Fleshing out the idea of my earlier comment,  should it strike your fancy:
Tests would look something like adding an additional test `Test_setHealthEndpointsDefaults` which ensures that a Function with no values in its `.HealthEndpoints` results in the LivenessProbe receiving the default values `DefaultLiveness` and `DeafaultReadiness`...  which are of course set with the standard Kube values.